### PR TITLE
Technical debt cleanup

### PR DESCRIPTION
**Pull request by ryanmelt**
remove notifications
add hostenter util script
fix mock_redis xread block
add metrics hook
add sleeper file handle close
fix anonymous redis volume

# Todo
Python updates

### CODE REVIEW

**Review by jmthomas — COMMENTED** [review]
I think this is a good simplification of the alert / notification system. For now we'll only put ERROR messages as alerts that must be acknowledged. Everything else goes in notifications. Future enhancement will allow users to customize which level goes where.

File: openc3/lib/openc3/utilities/logger.rb [hunk 1]
```diff
@@ -195,9 +195,7 @@ def log_message(severity_string, message, scope:, user:, type:, url:)
           if scope
             Topic.write_topic("#{scope}__openc3_log_messages", data)
           else
-            # The base openc3_log_messages doesn't have an associated logger
-            # so it must be limited to prevent unbounded stream growth
-            Topic.write_topic("openc3_log_messages", data, '*', 1000)
+            Topic.write_topic("NOSCOPE__openc3_log_messages", data)
```
Comment (jmthomas) [inline openc3/lib/openc3/utilities/logger.rb:198]: Do we have a good idea what is being logged without a scope?

Comment (ryanmelt) [reply openc3/lib/openc3/utilities/logger.rb:198]: yes.  Primarily API calls to the cmd-tlm-api.   Also some top level operator stuff.

File: openc3.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 set +e
 
 export DOCKER_COMPOSE_COMMAND="docker compose"
-${DOCKER_COMPOSE_COMMAND} version
+${DOCKER_COMPOSE_COMMAND} version &> /dev/null
```
Comment (jmthomas) [inline openc3.sh:6]: Should we pipe the .bat version to NUL?

File: openc3/lib/openc3/models/scope_model.rb [hunk 3]
```diff
@@ -32,6 +32,11 @@ class ScopeModel < Model
     PRIMARY_KEY = 'openc3_scopes'
 
     attr_accessor :children
+    attr_accessor :text_log_cycle_time
+    attr_accessor :text_log_cycle_size
+    attr_accessor :text_log_retain_time
```
Comment (jmthomas) [inline openc3/lib/openc3/models/scope_model.rb:37]: Shouldn't this be `tool_log_retain_time`

File: compose.yaml [hunk 4]
```diff
@@ -94,7 +94,7 @@ services:
   openc3-cosmos-cmd-tlm-api:
     # For rootless podman - Uncomment this user line and comment out the next
     # user: 0:0
-    user: "${OPENC3_USER_ID}:${OPENC3_GROUP_ID}"
+    user: "${OPENC3_USER_ID:-1000}:${OPENC3_GROUP_ID:-1000}"
```
Comment (jmthomas) [inline compose.yaml:97]: Why the change to hard code the user and group ids?

Comment (ryanmelt) [reply compose.yaml:97]: Its placing the default values of 1000 here instead of in the .env.  If openc3.sh sets the values that wins over the default.

File: openc3.sh (hunk 2 above)
Comment (ryanmelt) [reply openc3.sh:6]: We don't do this on Windows because we assume that we always have "docker compose"
